src/StubEnvironment.php

```diff
@@ -13,7 +14,7 @@ class StubEnvironment extends Environment
     /**
      * {@inheritdoc}
      */
-    public function getFilter($name)
+    public function getFilter(string $name): ?TwigFilter
```

Comment:
if I'm not mistaken, a signature like below would be compatible with both 2.x and 3.x of twig
```php
public function getFilter($name): ?TwigFilter
```

Reply:
Yes, this worked for all but `getTest`, I have added an if statement with a comment to clarify.